frontend/alert/components/Timeline/index.tsx

```diff
@@ -217,19 +233,7 @@ const Timeline = ({ courseCode, setTimeline }: TimelineProps) => {
     }, [courseCode]);
 
     return (
-        <AlertHistoryContainer
-            close={close}
-            ref={useScroll(
-                useOnClickOutside(
-                    () => {
-                        setTimeline(null);
-                        courseCode = null;
-                    },
-                    close,
-                    "historyIcon"
-                )
-            )}
-        >
+        <AlertHistoryContainer close={close} ref={onScroll(onClickOutside)}>
```

Comment:
This is still a hook, the alias makes it a bit confusing, I'd move it up to `const scrollRef = useOnScroll(onClickOutside)` or something like that

Reply:
Ohh got it!